finalPrep/modZeroHero.js

```diff
@@ -1,84 +1,151 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Hulk";
+var specialAbility = "Super strength";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "Look it's ";
+console.log(greeting + heroName + "!");
+var catchPhrase = "I will save you all";
+console.log(`${catchPhrase} with my ${specialAbility}`);
 // Declare two variables - power AND energy - set to integers
-
+var power = 3500
+var energy = 3250
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power * 500;
+console.log(fullPower);
+var fullEnergy = energy +150;
+console.log(fullEnergy);
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = true;
+console.log(isHuman);
+var identityConcealed = true;
+console.log(identityConcealed);
 
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Thanos", "Red Hulk", "Abomination"];
+console.log(archEnemies);
+var sidekicks = ["Rick Jones", "Abigail Brand", "Adam Warlock"]
+console.log(sidekicks);
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push("Wendigo");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  if (dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if (dangerLevel >= 10){
```

Comment:
Order of conditions does matter, so this works, however I will point out that it is not best practice to have conditions that overlap... in this case, numbers that are over 50 are also over 10. While the order of the conditions makes it so that this works fine, I suggest putting the more complex logic in your `else` block.